src/main/zapHomeFiles/hud/target/inject.js

```diff
@@ -215,7 +202,7 @@
 		} else {
 			// Count the number of hidden fields
 			var inputs = document.getElementsByTagName('input');
-			for (index = 0; index < inputs.length; ++index) {
+			for (var index = 0; index < inputs.length; ++index) {
```

Comment:
should this be `let`?

Reply:
yeah, will change that before I merge...